README.md

```diff
@@ -30,4 +43,42 @@ Let's say Alice wants to swap 0.25 ETH into Stellar, and Bob wants to swap 0.25
 
 Additionally, both parties can refund their part of the swap after a certain (long) time period. This prevents malicious actors from stealing money, and other errors that would cause swaps to fail. For security reasons, the swap initiator (Bob) will have to wait longer to claim their refund than the swap fulfiller. Read more here: https://blog.lightning.engineering/announcement/2017/11/16/ln-swap.html
 
-Please let me know if I have overlooked anything, or if you have suggestions. Thanks!
\ No newline at end of file
+Please let me know if I have overlooked anything, or if you have suggestions. Thanks!
+
+
+Setup
+============
+
+## Prerequisites
+* `Docker`
+* `docker-compose`
+* `yarn`
+* `npm`
+
+
+## Environment Setup
+```bash
+# Clone the project
+git clone https://github.com/mvanderh/MirrorX.git
+cd MirrorX
+
+# Download Docker dependencies
+docker-compose build
+
+# Install yarn dependencies
+yarn install
+
+# Install client dependencies
+cd www
+yarn install
+npm install
```

Comment:
Don't need to `npm install` after `yarn install`

Reply:
Should we consider killing or amending: https://github.com/jonahglover/MirrorX/blob/master/www/README.md

as part of this PR?